Automattic/jetpack
# Sharing: add Threads service

Fixes #33316

## Proposed changes:

This adds both the sharing button and the sharing button block.

### Other information:


## Jetpack product discussion

No

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Go to Appearance > Themes
2. Install and activate the Twenty Ten theme
3. Go to wordpress.com/marketing/sharing-buttons/
4. Add a Bluesky sharing button to your site (note that https://github.com/Automattic/wp-calypso/pull/88239 and https://github.com/Automattic/jetpack/pull/36227 need to be merged first)
5. After saving, it should look good on your site.
6. Try using different button styles (official, icon, icon+text, text)
7. Next, go to Appearance > Themes and switch to a block-based theme like Twenty Twenty Four
8. Go to Jetpack > Settings > Sharing and disable the sharing feature.
9. Go to Apperance > Site Editor > Templates > Single Posts
10. Add a sharing buttons block to your template, and add a Threads  button.
11. Ensure it is displayed properly on the frontend, and that the button works well.
